src/pages/landing-page/components/find-out/find-out.component.tsx

```diff
@@ -10,13 +10,13 @@ export const FindOut: React.FC = () => {
   return (
     <div className={styles.block}>
       <h2 className={styles.block__title}>Дізнайтеся чи правильно ви доглядаєте за соїм улюбленцем</h2>
-      <img src={img1} width="200" className={styles.block__img}></img>
+      <img src={img1} width="200" className={styles.block__img} alt="Img"></img>
```

Comment:
Not sure if Img is a good value for an alt attribute. Would you like to add something more meaningful?

Reply:
I'will fix it